dashboard/src/app/(authenticated)/components/Search.tsx

```diff
@@ -35,7 +35,7 @@ export const Search: FC<{}> = () => {
           <RefreshCwIcon className="h-8 w-8 animate-spin text-blue-500" />
         </div>
       ) : (
-        <div className="min-h-[360px]">
```

Comment:
Any reason why to remove the min height. I believe it was there so the footer with the "data per load" select would be far below if there was only one WfSpec

Reply:
I removed the `min-h-[360px]` to bring the load footer closer to the end of the `WfSpec` list for these reasons:

- Less mouse travel time
- More visibly obvious to the user
- Less white space looks cleaner

Even with just one `WfSpec,` I'm not sure how a distanced footer would be more beneficial. I'm always open to hear other's thoughts, so please let me know what you think.

Reply:
No that make sense I was just curious why the change but what you said tracks. 😎

Reply:
The default space is there for view consistency user can predict where the position of elements would be so I would prefer to keep it.